### DISCUSSION TITLE
我在ue5引擎上使用sherpa-ncnn实现麦克风语音实时转换成文字的功能，我希望能分享我的虚化引擎插件

### DISCUSSION THREAD

**Discussion by XPDD** [k2-fsa/sherpa#641] [Show and tell]
博客链接 https://blog.csdn.net/weixin_45335638/article/details/141562837?spm=1001.2014.3001.5502

**Comment by XPDD** [comment]
非常感谢sherpa-ncnn，让我们的麦克风输入可以实时转换成文字。当然还有很多挑战，比如使用hotwords_file时会ue引擎会闪退等，还有在ue子线程运行时，识别效果不理想等。

**Comment by csukuangfj** [comment]
优秀！


是否打算在 github 上开源代码呢？

**Reply by XPDD** [reply]
由于是Vc++工程，等到迁移到Cmake工程会开放代码

**Reply by csukuangfj** [reply]
> 由于是Vc++工程，等到迁移到Cmake工程会开放代码

期待

**Comment by XPDD** [comment]
![image](https://github.com/user-attachments/assets/d1050203-2427-4f85-880d-ac7bb13aa0e6)
我配置了热词，但是识别效果不是很好。请问热词是这样用的吗？
![image](https://github.com/user-attachments/assets/5ac16a95-1c3d-473f-b735-e5129f4d304b)

**Reply by XPDD** [reply]
![image](https://github.com/user-attachments/assets/c9c19045-d6ad-499c-81a8-404e4c6b3304)

**Comment by csukuangfj** [comment]
不对。 每一行的词，字与字之间，要用空格隔开。

https://github.com/k2-fsa/sherpa-ncnn/blob/master/.github/scripts/run-test.sh#L68
这里有例子，你自己去看一下？

**Reply by XPDD** [reply]
谢谢！我找了很久。
